src/main/scala/is/hail/expr/ir/Compile.scala

```diff
@@ -40,7 +40,7 @@ object Compile {
     val fb = new EmitFunctionBuilder[F](argTypeInfo, GenericTypeInfo[R]())
 
     var ir = body
-    ir = Optimize(ir)
+    ir = Optimize(ir, noisy = false)
```

Comment:
should we also modify MT and Table `.value`? and `Interpret`?

Reply:
no, we want it in both of those, I think. Really this is just to prevent us from being noisy AGAIN inside every `Compile` call